db/migrate/20231122100839_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :body
+      t.references :author, null: false, foreign_key: { to_table: :users }
+      t.integer :comments_counter, default: 0
+      t.integer :likes_counter, default: 0
+
```

Comment:
- [ ] in the post table there should be a column for a photo as well according to the requirements.

Reply:
Hi @mikethreels 
By following the ERD, the column for a photo is not in the posts table its in `users`. 
Can you check again and elaborate more?


![image](https://github.com/GhausAliKhan/Blog-app/assets/94997336/cdc3c399-9036-440a-9e4c-27496745f78f)